pipeline-runner/R/gem2s-7-upload_to_aws.R

```diff
@@ -1,19 +1,20 @@
 
 upload_to_aws <- function(input, pipeline_config, prev_out) {
   message("Uploading to AWS ...")
-  check_names <- c("config", "counts_list", "annot", "doublet_scores", "scdata_list", "scdata", "qc_config")
+  check_names <- c("config", "counts_list", "annot", "doublet_scores", "scdata_list", "qc_config")
   check_prev_out(prev_out, check_names)
 
   experiment_id <- input$experimentId
   project_id <- input$projectId
 
   # destructure what need from prev_out
-  scdata <- prev_out$scdata
+  scdata_list <- prev_out$scdata_list
   config <- prev_out$config
   config_dataProcessing <- prev_out$qc_config
```

Comment:
why don't you name `config_dataProcessing` to `qc_config` to match the name in `prev_out`?